workers/unity/Assets/Fps/Scripts/Config/MapBuilder.cs

```diff
@@ -344,14 +361,14 @@ private void PlaceTile(Vector2Int tileCoord)
 
         private void PlaceTile(Vector2Int tileCoord, GameObject tile, float rotation)
         {
-            var tileOffset = UnitsPerTile / 2;
+            var tileOffset = mapBuilderSettings.UnitsPerTile / 2;
```

Comment:
this throws a nullpointer exceptton if you don't set `mapBuilderSettings` before hitting `Generate Map`. should have a check for that in the beginning